joern-cli/frontends/pysrc2cpg/src/test/scala/io/joern/pysrc2cpg/dataflow/DataFlowTests.scala

```diff
@@ -20,7 +19,7 @@ class DataFlowTests extends PySrc2CpgFixture(withOssDataflow = true) {
       |""".stripMargin)
     val source = cpg.literal("42")
     val sink   = cpg.call.code("print.*").argument
-    sink.reachableByFlows(source).size shouldBe 1
+    sink.reachableByFlows(source).size shouldBe 2
```

Comment:
This and the flows below are simply introduced by the extra available steps within the de-sugared block